src/client/dfuse/pil4dfs/hook.c

```diff
@@ -639,6 +731,25 @@ determine_mem_block_size(const void *addr)
 	}
 }
 
+void
+free_memory_in_hook(void)
+{
+	int i;
+
+	D_FREE(path_ld);
+	D_FREE(path_libc);
+	D_FREE(module_list);
+	if (path_libpthread)
+		free(path_libpthread);
```

Comment:
minor, but free() is fine with NULL.
```suggestion
	free(path_libpthread);
```